### PR TITLE
Added Fees and Volume adapter for DerpDEX

### PR DESCRIPTION
**Pull request by techyderp**
Added Fees and Volume adapter for DerpDEX.
Already added DerpDEX to DefiLlama here - https://github.com/DefiLlama/DefiLlama-Adapters/pull/6815

### CODE REVIEW

**Comment by llamatester** [comment]
The DerpDEX adapter exports: 
        
 
 _______________________________________
    Dexs for 17/7/2023
    _______________________________________
    
    ERA 👇
    Backfill start time: 1/4/1970
    NO METHODOLOGY SPECIFIED
    Total volume: 125857.81489427376
    Daily volume: 15000.52294670308
    Timestamp: 1689552000
    
    
    
    
    Done in 3.70s.

**Comment by llamatester** [comment]
The DerpDEX.ts adapter exports: 
        
 
 _______________________________________
    Fees for 17/7/2023
    _______________________________________
    
    ERA 👇
    Backfill start time: 1/4/1970
    Timestamp: 1689638398
    Daily fees: 228.2575032154379
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 32.728493484694525
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 228.2575032154379
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 32.728493484694525
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    
    
    Done in 5.56s.

File: dexs/DerpDEX/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { fetchVolume } from "./DerpDEX";
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    [CHAIN.ERA]: {
+      fetch: fetchVolume(CHAIN.ERA),
+      start: async () => 7790768,
```
Comment (dtmkeng) [inline dexs/DerpDEX/index.ts:9]: could your fix start time to this `1688515200`

File: fees/DerpDEX.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Fees: "Fees generated on each swap at a rate set by the pool.",
+  TotalUserFees: "Cumulative all-time Fees",
+  TotalFees: "Cumulative all-time Fees",
+};
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    [CHAIN.ERA]: {
+      fetch: fetchFee(CHAIN.ERA),
+      start: async () => 7790768,
```
Comment (dtmkeng) [inline fees/DerpDEX.ts:16]: could your fix start time to this `1688515200`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Pull request merged**